msrest/pipeline/universal.py

```diff
@@ -45,6 +46,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+BOM = codecs.BOM_UTF8.decode(encoding='utf-8')
```

Comment:
`_BOM` please, I like to keep my stuff private :)

Reply:
Last tiny detail and I merge it ;)

Reply:
you are completely right!